.github/dependency-review-config.yml

```diff
@@ -0,0 +1,11 @@
+
+allow-licenses:
+  - Apache-2.0
+  - MIT
+  - BSD-3-Clause
+  - BSD-2-Clause
+  - MPL-2.0
+  - HPND
+  - PSF-2.0
+
+fail-on-severity: low
```

Comment:
what does that mean?

Reply:
It defines the threshold for level of severity (low, moderate, high, critical).
The action will fail on any pull requests that introduce vulnerabilities of the specified severity level or higher.